Automattic/pocket-casts-ios
# [Paywall Experiment] Reviews variant

| 📘 Part of: #1986
|:---:|

Fixes #2031

This PR introduces the last Paywall variant: Reviews. We want to display a list of Reviews.

https://github.com/user-attachments/assets/c333a026-1e5f-4758-9f0e-baed34aa7f25

## To test

> [!NOTE]
> Make sure to enable the upgradeExperiment feature flag

- CI must be green 🟢 
- Use a non-plus account
- Go to `Profile -> Settings` and tap on `Pocket Casts Plus`
- Check the new paywall layout like the video above

## Checklist
